src/hooks/force-update.ts

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from 'react';
+import { Linking, Platform } from 'react-native';
+import DeviceInfo from 'react-native-device-info';
+
+import remoteConfig from '@react-native-firebase/remote-config';
+
+export const useValidateAppVersion = () => {
+  const [showModal, setShowModal] = useState(false);
+
+  useEffect(() => {
+    const checkAndUpdate = async () => {
+      const updated = await isUpdated();
+      if (!updated) {
+        setShowModal(true);
+      }
+    };
+    checkAndUpdate();
+  });
+
+  return { showModal, openStore };
+};
+
+const isUpdated = async (): Promise<boolean> => {
+  remoteConfig().setDefaults({
+    minimum_version: '',
+  });
+
+  const fetchedRemotely = await remoteConfig().fetchAndActivate();
+
+  if (fetchedRemotely) {
+    const remoteVersion = remoteConfig().getValue('').asString();
+    const localVersion = DeviceInfo.getVersion();
+    return compareVersions(localVersion, remoteVersion);
+  }
+  return true;
+};
+
+const openStore = () => {
+  Platform.select({
+    ios: Linking.openURL(''),
+    android: Linking.openURL(''),
```

Comment:
You can add these values to the env file and use them with the `react-native-config` library

```suggestion
    ios: Linking.openURL(APP_STORE_URL),
    android: Linking.openURL(PLAY_STORE_URL),
```

Reply:
Done